src/ConfigProvider.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\I18n\PhoneNumber;
+
+use Laminas\ServiceManager\ConfigInterface;
+
+/**
+ * @see ConfigInterface
+ *
+ * @psalm-import-type ServiceManagerConfigurationType from ConfigInterface
+ *
+ * @psalm-type PackageConfig = array{
+ *     default-country-code?: non-empty-string|null
+ * }
+ */
+final class ConfigProvider
+{
+    public function __invoke(): array
+    {
+        return [
+            'laminas-i18n-phone-number' => [
+                'default-country-code'    => null,
+                'acceptable-number-types' => PhoneNumberValue::TYPE_RECOMMENDED,
```

Comment:
This is the only use of `TYPE_RECOMMENDED`, theoretically making it easy to override what is considered an acceptable number type in a place that is easy to document